packages/cli/test/acceptance/smoke.acceptance.test.ts

```diff
@@ -7,6 +7,14 @@ import * as qq from 'qqjs'
 
 import commandsOutput from './commands-output'
 
+const stripAnsi = (input: string) => {
+  // eslint-disable-next-line no-control-regex, unicorn/escape-case
+  const ansiRegex = /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]|\s|─/g
+  const cleanedString = input.replace(ansiRegex, '')
+
+  return cleanedString
+}
+
```

Comment:
Is there a reason to do this instead of using the `strip-ansi` package we have already in our dependencies?

Reply:
The regex pattern also includes whitespace and other characters that are not included in the `stripAnsi` package. This is a custom string cleaning function. I think this will be better than running 2 operations on the input string with `stripAnsi` and then replacing other characters outside of the `stripAnsi` regex.